cloud/main.js

```diff
@@ -118,3 +131,63 @@ Parse.Cloud.afterSave("Game", function(request) {
         }
     });
 });
+
+/*
+ * For non-provisional players, adjust the player's rating based on
+ * the game outcome and the opponent's rating.
+ * https://en.wikipedia.org/wiki/Elo_rating_system#Mathematical_details
+ */
+function computeEloRating(player, opponentRating, playerWon) {
+    /*
+     * Assuming 0 points for a loss and 1 for a win, compute the
+     * expected (average) score for this player against this
+     * opponent.
+     */
+
+    var rating = player.get("rating");
+    var deltaRating = opponentRating - rating;
+   
+    var ln10over400 = 5.76e-3;
+    var expected = 1/(1 + Math.exp(ln10over400 * deltaRating));
+    var actual = playerWon ? 1 : 0;
+
+    var K = 32; // arbitrary constant
```

Comment:
If the two players are equally rated, the rating change will be half this value. That means 16 points at stake for two non-provisional players of the same rating. This could be adjusted to be a rounder number, like 20 (K = 40).